Tasks/words.js

```diff
@@ -1,26 +1,27 @@
 // Count words
+'use strict';
 
- Words=function(s) {
- number_of_words_in_s = 0
- flag = false
- for (c of s) {
-   if (!flag) {
-   if (c === ' ') {
-   if (flag === true) { flag = false } else { flag = false }
+const countWords = (str) => {
+  let wordsNum = 0;
+  let flag = false;
+  for (const element of str) {
+    if (!flag) {
+      if (element === ' ') {
+        flag = false;
+      } else {
+        flag = true;
+      }
```

Comment:
Try to optimize this

Reply:
Okay, you can see the changings in a new commit.